src/test/java/org/gridsuite/voltageinit/server/VoltageInitControllerTest.java

```diff
@@ -105,18 +102,19 @@ public class VoltageInitControllerTest {
     private final RestTemplateConfig restTemplateConfig = new RestTemplateConfig();
     private final ObjectMapper mapper = restTemplateConfig.objectMapper();
 
-    private Network network;
+    private Network network = EurostagTutorialExample1Factory.createWithMoreGenerators(new NetworkFactoryImpl());
     private Network network1;
     private Network networkForMergingView;
     private Network otherNetworkForMergingView;
-    CompletableFutureTask<OpenReacResult> completableFutureResultsTask = CompletableFutureTask.runAsync(() -> RESULT, ForkJoinPool.commonPool());
+    OpenReacParameters openReacParameters = new OpenReacParameters();
+    OpenReacResult openReacResult = new OpenReacResult(OpenReacStatus.OK, new OpenReacAmplIOFiles(openReacParameters, network, false), INDICATORS);
```

Comment:
Same, depending how time to you need it (so in one method or several), you can init it in the setup or create it on the fly when needed

Reply:
Done, moved in the setUp